### ISSUE TITLE
no-standalone-expect rule triggered when calling assertions in expect.extend blocks for custom matchers

### ISSUE THREAD

**Issue by nickhall** [playwright-community/eslint-plugin-playwright#319]
When writing custom matchers using [the code provided by the docs](https://playwright.dev/docs/test-assertions#add-custom-matchers-using-expectextend), the calls to `baseExtend` are flagged by the `no-standalone-expect` rule.

```ts
import { expect as baseExpect } from '@playwright/test';

const expect = baseExpect.extend({
  async someMatcher() {
    // triggers the error
    await baseExpect(true).toBeTruthy();
  }
}
```

I would expect that this scenario wouldn't trigger the rule.

**Comment by carlbray** [comment]
You might need to declare aliases 
https://github.com/playwright-community/eslint-plugin-playwright/blob/main/README.md#aliased-playwright-globals

```json
{
  "settings": {
    "playwright": {
      "globalAliases": {
        "test": ["myTest"],
        "expect": ["myExpect"]
      }
    }
  }
}
```

**Comment by nickhall** [comment]
Thank you, I think that should handle this use case.
